src/sounddeviceportaudio.cpp

```diff
@@ -752,12 +752,13 @@ int SoundDevicePortAudio::callbackProcess(const unsigned int framesPerBuffer,
 }
 
 int SoundDevicePortAudio::callbackProcessClkRef(const unsigned int framesPerBuffer,
-                                          CSAMPLE *out, const CSAMPLE *in,
-                                          const PaStreamCallbackTimeInfo *timeInfo,
-                                          PaStreamCallbackFlags statusFlags) {
+                                                CSAMPLE *out, const CSAMPLE *in,
+                                                const PaStreamCallbackTimeInfo *timeInfo,
+                                                PaStreamCallbackFlags statusFlags) {
     PerformanceTimer timer;
     timer.start();
-    Trace trace("SoundDevicePortAudio::callbackProcessClkRef " + getInternalName());
+
+    Trace trace("SoundDevicePortAudio::callbackProcessClkRef %1", getInternalName());
```

Comment:
If developer mode is active then the trace does re-build the key every time. If developer mode is off then no QString conversions or appends happen (they stay const char*'s and Trace::initialize doesn't get called).